src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public static final int MAX_NUMBER = 101;
+    private final Random random = new Random(47);
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Ball getRandomBall() {
```

Comment:
Don't create temporary variables if you can directly return value from the method. You can simplify the method by returning the new `Ball` instance directly.